internal/rbd/errors/errors.go

```diff
@@ -67,4 +67,8 @@ var (
 	ErrGroupNotConnected = fmt.Errorf("%w: RBD group is not connected", librados.ErrNotConnected)
 	// ErrGroupNotFound is returned when group is not found in the cluster on the given pool and/or namespace.
 	ErrGroupNotFound = fmt.Errorf("%w: RBD group not found", librbd.ErrNotFound)
+	// ErrGroupUnavailable is returned when the requested group is unavailable.
+	ErrGroupUnavailable = errors.New("rbd group is unavailable")
```

Comment:
Can you add more info here

similar to the info documented for ErrUnavailable above
```
	// ErrUnavailable is returned when the image needs to be recreated
	// locally and may be corrected by retrying with a backoff.
	ErrUnavailable = errors.New("image needs to be recreated")
```

maybe keep the error message very similar too

Reply:
Done